src/main/java/dao/CartDao.java

```diff
@@ -0,0 +1,63 @@
+package dao;
+
+import entity.Cart;
+import entity.Product;
+import mapper.CartMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.Objects;
+
+@Repository
+public class CartDao {
+    private JdbcTemplate jdbcTemplate;
+    private CartMapper cartMapper;
+
+    public CartDao(final JdbcTemplate jdbcTemplate, final CartMapper cartMapper) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.cartMapper = cartMapper;
+    }
+
+    public Cart createCart() {
+        jdbcTemplate.update("INSERT INTO Carts DEFAULT VALUES");
```

Comment:
Put all queries you specified here into `SQLQuery` class as you did for some queries.